proto/auth/auth.proto

```
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package auth;
```

Buf (1.55.1)

Files with package "auth" must be within a directory "auth" relative to root but were in directory "proto/auth".

(PACKAGE_DIRECTORY_MATCH)